users/tests/test_user_login.py

```diff
@@ -22,3 +33,20 @@ def test_user_client_should_not_log_in_when_email_is_none(self):
 
     def test_user_client_should_not_log_in_when_password_is_none(self):
         self.assertFalse(self.client.login(email="testuser@codepoets.it", password=None))
+
+    def test_user_should_redirect_to_change_password_form_if_need_change_password_is_true(self):
+        data = {"username": self.user.email, "password": self.password}
```

Comment:
At the beginning of this test you can add additional assertion to ensure that `need_change_password` user property is set on `True`

Reply:
Done.